PHPCompatibility/Tests/FunctionUse/NewFunctionsUnitTest.php

```diff
@@ -64,1030 +60,1031 @@ public function testNewFunction($functionName, $lastVersionBefore, $lines, $okVe
-            ['intltz_has_same_rules', '5.4', [271], '5.5'],
-            ['intltz_get_display_name', '5.4', [272], '5.5'],
-            ['intltz_get_dst_savings', '5.4', [273], '5.5'],
-            ['intltz_from_date_time_zone', '5.4', [274], '5.5'],
-            ['intltz_to_date_time_zone', '5.4', [275], '5.5'],
-            ['intltz_get_error_code', '5.4', [276], '5.5'],
-            ['intltz_get_error_message', '5.4', [277], '5.5'],
-            ['opcache_get_configuration', '5.4', [490], '5.5'],
-            ['opcache_get_status', '5.4', [491], '5.5'],
-            ['opcache_invalidate', '5.4', [492], '5.5'],
-            ['opcache_reset', '5.4', [493], '5.5'],
+            ['array_column', '5.4', 170, '5.5'],
+            ['boolval', '5.4', 171, '5.5'],
+            ['json_last_error_msg', '5.4', 172, '5.5'],
+            ['password_get_info', '5.4', 173, '5.5'],
+            ['password_hash', '5.4', 174, '5.5'],
+            ['password_needs_rehash', '5.4', 175, '5.5'],
+            ['password_verify', '5.4', 176, '5.5'],
+            ['hash_pbkdf2', '5.4', 177, '5.5'],
+            ['openssl_pbkdf2', '5.4', 178, '5.5'],
+            ['curl_escape', '5.4', 179, '5.5'],
+            ['curl_file_create', '5.4', 180, '5.5'],
+            ['curl_multi_setopt', '5.4', 181, '5.5'],
+            ['curl_multi_strerror', '5.4', 182, '5.5'],
+            ['curl_pause', '5.4', 183, '5.5'],
+            ['curl_reset', '5.4', 184, '5.5'],
+            ['curl_share_close', '5.4', 185, '5.5'],
+            ['curl_share_init', '5.4', 186, '5.5'],
+            ['curl_share_setopt', '5.4', 187, '5.5'],
+            ['curl_strerror', '5.4', 188, '5.5'],
+            ['curl_unescape', '5.4', 189, '5.5'],
+            ['date_create_immutable_from_format', '5.4', 544, '5.5'],
+            ['date_create_immutable', '5.4', 545, '5.5'],
+            ['imageaffine', '5.4', 682, '5.5'],
+            ['imageaffinematrixconcat', '5.4', 190, '5.5'],
+            ['imageaffinematrixget', '5.4', 191, '5.5'],
+            ['imagecrop', '5.4', 192, '5.5'],
+            ['imagecropauto', '5.4', 193, '5.5'],
+            ['imageflip', '5.4', 194, '5.5'],
+            ['imagepalettetotruecolor', '5.4', 195, '5.5'],
+            ['imagescale', '5.4', 196, '5.5'],
+            ['imagesetinterpolation', '5.4', 685, '5.5'],
+            ['mysqli_begin_transaction', '5.4', 197, '5.5'],
+            ['mysqli_release_savepoint', '5.4', 198, '5.5'],
+            ['mysqli_savepoint', '5.4', 199, '5.5'],
+            ['socket_sendmsg', '5.4', 202, '5.5'],
+            ['socket_recvmsg', '5.4', 203, '5.5'],
+            ['socket_cmsg_space', '5.4', 204, '5.5'],
+            ['cli_get_process_title', '5.4', 205, '5.5'],
+            ['cli_set_process_title', '5.4', 206, '5.5'],
+            ['datefmt_format_object', '5.4', 207, '5.5'],
+            ['datefmt_get_calendar_object', '5.4', 208, '5.5'],
+            ['datefmt_get_calendar_object', '5.4', 211, '5.5'],
```

Comment:
I realize this comes from the original test array, but looking at the test case file, I don't see any difference between the two tests, so I think one of these entries can be safely removed from both the test case file + this file.

In the test case file, turning it into an empty line is fine to prevent having to renumber all the tests after the function call being removed.